src/nrc/datamodel/models.py

```diff
@@ -179,3 +180,80 @@ class NotificatieResponse(models.Model):
 
     def __str__(self) -> str:
         return f"{self.abonnement} {self.response_status or self.exception}"
+
+
+class CloudEvent(models.Model):
+    int_id = models.AutoField(
+        primary_key=True, serialize=False, verbose_name="ID", help_text=_("internal id")
+    )
+
+    id = models.CharField(_("id"), max_length=255, help_text=_("event id"))
+    source = models.CharField(_("source"), max_length=255, help_text=_("event source"))
+    specversion = models.CharField(
+        _("specversion"),
+        max_length=255,
+        validators=[RegexValidator(regex=r"^(\d+)\.(\d+)")],
+        help_text=_("cloudevent spec version used by the event"),
+    )
+    type = models.CharField(
+        _("type"),
+        max_length=255,
+        help_text=_(
+            "event type on which can be subscribed. Example: nl.overheid.zaken.zaak.created"
+        ),
+    )
+
+    datacontenttype = models.CharField(
+        _("datacontent"),
+        max_length=255,
+        blank=True,
+        help_text=_(
+            "content type of the (optional) cloudevent data. Example: application/json"
+        ),
+    )
+    dataschema = models.CharField(
+        _("dataschema"),
+        max_length=255,
+        blank=True,
+        help_text=_("A schema that the data must follow"),
+    )  # TODO needed?
```

Comment:
I guess we won't really be using it for now, but let's keep it since it's part of the spec